db/schema.rb

```diff
@@ -37,6 +37,16 @@
   add_index "categories", ["name"], name: "index_categories_on_name", using: :btree
   add_index "categories", ["user_id"], name: "index_categories_on_user_id", using: :btree
 
+  create_table "photos", force: :cascade do |t|
+    t.integer  "imageable_id"
+    t.string   "imageable_type"
+    t.datetime "created_at",     null: false
+    t.datetime "updated_at",     null: false
+    t.string   "file"
+  end
+
+  add_index "photos", ["imageable_type", "imageable_id"], name: "index_photos_on_imageable_type_and_imageable_id", using: :btree
+
   create_table "transactions", force: :cascade do |t|
```

Comment:
Very nice that you pushed your own photos to repo, but please remove them. And add them to .gitignore